schemas/Product.yaml

```diff
@@ -1,20 +1,154 @@
-required:
+# copyright 2021 Lucid Green, Inc.
+
+required: 
   - id
   - name
-  - strainId
-  - strainName
-  - productCategoryName
-  - productCategoryType
-  - quantityType
-  - publicIngredients
-  - description
-  - isUsed
+  - updated_on
```

Comment:
I don't think updated_on should be required.

Reply:
agreed.